libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookHelper.cs

```diff
@@ -167,6 +174,19 @@ public static List<Attachment> HandleMessageAttachments(Message message)
             return attachmentsList;
         }
 
+        /// <summary>
+        /// Sends a message that will display a typing indicator as a placeholder while the message arrives.
+        /// </summary>
+        /// <param name="activity">The activity with the actual message to be sent.</param>
+        /// <param name="turnContext">The TurnContext that will send the activities.</param>
+        /// <param name="cancellationToken">A cancellation token for the task.</param>
+        /// <returns>A task.</returns>
+        public static async Task SendMessage(IActivity activity, ITurnContext turnContext, CancellationToken cancellationToken)
```

Comment:
I would change the name of this method. Something like 'SendMessageWithTypingAsync' or similar. Don't forget the Async in the end.